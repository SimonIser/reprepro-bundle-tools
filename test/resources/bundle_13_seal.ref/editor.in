# Note: clean this file completely to CANCEL this current 'seal' action

= BUNDLE-METADATA =
Bundlename: mybionic/0001
BasedOn: NEW
Distribution: mybionic
Target: plus
Creator: chlu

= RELEASENOTES =
This is my best test bundle

<Details>

__DYNAMIC_PACKAGE_LIST__

