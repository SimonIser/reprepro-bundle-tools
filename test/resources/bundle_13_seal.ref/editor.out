# Note: clean this file completely to CANCEL this current 'seal' action

= BUNDLE-METADATA =
Bundlename: mybionic/0001
BasedOn: NEW
Distribution: mybionic
Target: plus
Creator: chlu

= RELEASENOTES =
This is my best test bundle

We just need this bundle for the test automation.

__DYNAMIC_PACKAGE_LIST__

